modules/UI/videolayout/VideoLayout.js

```diff
@@ -859,6 +859,20 @@ var VideoLayout = {
         }
 
         VideoLayout.resizeThumbnails();
+
+        const remoteVideosCount = Object.keys(remoteVideos).length;
+
+        if (!remoteVideosCount) {
+            window.setTimeout(() => {
+                const updatedRemoteVideosCount
+                    = Object.keys(remoteVideos).length;
+
+                if (!updatedRemoteVideosCount) {
+                    this._maybePlaceParticipantOnLargeVideo(
+                        APP.conference.getMyUserId());
+                }
+            }, 3000);
+        }
```

Comment:
We actually have a function that takes care of that: VideoLayout.updateAfterThumbRemoved(). I think we should fix it inside this function.